functions/utils/getToken.ts

```diff
@@ -11,12 +12,30 @@ function formatTitleName(symbol: string, name: string) {
   return 'View Token on Uniswap'
 }
 
-export default async function getToken(networkName: string, tokenAddress: string | undefined, url: string) {
+const convertTokenAddress = (tokenAddress: string, networkName: string) => {
```

Comment:
It's confusing to switch the order of arguments - prefer to match `getToken`:

```suggestion
const convertTokenAddress = (networkName: string, tokenAddress: string) => {
```